### DISCUSSION TITLE
Nodebox + self hosted bundler

### DISCUSSION THREAD

**Discussion by jfmmm** [codesandbox/sandpack#831] [Q&A]
Hi,

Can you use Nodebox with a self hosted bundler?

I got a normal sandbox to work with it but the nodebox one won't work. 

The second I switch to not using my bundler it work.

Not getting any error and I saw that you mentioned it's not open source yet. So I assume it's a no? just want to confirm.

Beside that I assume there no way to consume private package with it?

**Comment by danilowoz** [comment]
Maybe @DeMoorJasper can help you to figure this out

**Reply by DeMoorJasper** [reply]
We haven't implemented private package support for nodebox yet.

For self hosting you can contact us at sales@codesandbox.io
